<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dial_AsmReg</class>
 <widget class="QDialog" name="Dial_AsmReg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Read/Write Data</string>
  </property>
  <widget class="QTextEdit" name="textEdit_Read">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>131</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Select">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Asm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Address">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Address</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Length">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Length of Data</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Write">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>131</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>341</width>
     <height>531</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Read Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Write Data</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Hexa">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Hexadecimal</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Avtived">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Actived</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_FE">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Asm FrontEnd Nb</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>440</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Write">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Read">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Msg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>511</width>
     <height>79</height>
    </rect>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>textEdit_Read</zorder>
  <zorder>spinBox_Select</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit_Address</zorder>
  <zorder>label_2</zorder>
  <zorder>spinBox_Length</zorder>
  <zorder>label_3</zorder>
  <zorder>textEdit_Write</zorder>
  <zorder>pushButton_Close</zorder>
  <zorder>pushButton_Write</zorder>
  <zorder>pushButton_Read</zorder>
  <zorder>textEdit_Msg</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
